### PR TITLE
added IdentityConcentratorId

### PR DESCRIPTION
**Pull request by dvoet**
https://broadworkbench.atlassian.net/browse/CA-681

**PR checklist**
- [x] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [x] Bump the version in `project/Settings.scala` `createVersion()`
- [x] Update `CHANGELOG.md` for those libraries
- [x] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [x] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [x] Get two thumbsworth of PR review
- [x] Verify all tests go green (CI _and_ coverage tests)
- [x] Squash commits and **merge to develop**
- [x] Delete branch after merge

### CODE REVIEW

File: model/src/main/scala/org/broadinstitute/dsde/workbench/model/WorkbenchIdentity.scala [hunk 1]
```diff
@@ -21,7 +22,8 @@ sealed trait WorkbenchSubject
 
 final case class WorkbenchEmail(value: String) extends ValueObject
 final case class GoogleSubjectId(value: String) extends ValueObject
-final case class WorkbenchUser(id: WorkbenchUserId, googleSubjectId: Option[GoogleSubjectId], email: WorkbenchEmail)
+final case class IdentityConcentratorId(value: String) extends ValueObject
+final case class WorkbenchUser(id: WorkbenchUserId, googleSubjectId: Option[GoogleSubjectId], email: WorkbenchEmail, identityConcentratorId: Option[IdentityConcentratorId])
```
Comment (dvoet) [inline model/src/main/scala/org/broadinstitute/dsde/workbench/model/WorkbenchIdentity.scala:26]: I realize I could have given `identityConentratorId` a default to preserve backwards compatibility but I don't think this change should slip under the radar.

Comment (Qi77Qi) [reply model/src/main/scala/org/broadinstitute/dsde/workbench/model/WorkbenchIdentity.scala:26]: what is this field for?

Comment (yonghaoy) [reply model/src/main/scala/org/broadinstitute/dsde/workbench/model/WorkbenchIdentity.scala:26]: This is related with Terra Kernel's new authN system. We are standing up a service called identityConcentrator which can issue access token and some Ga4GH stuff: 
https://docs.google.com/document/d/1vIZAgxyR9xK3CaFCix4tG8aNkYqrVD1i2vAjOLLB-Ck/edit
And 
https://docs.google.com/document/d/1UsmomDFsmctpL7c3T7986N_AiyQBNJJCNo7bWB1oIBA/edit
BTW, doug, you have a typo here: identityConentratorId -> identityCon*c*entratorId

**Review by yonghaoy — APPROVED** [review]
👍👍

**Review by Qi77Qi — APPROVED** [review]

**Pull request merged**